backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/CassandraCluster.java

```diff
@@ -67,15 +57,19 @@ private CassandraCluster(CassandraModule module, @Named("cassandraHost") String
                 .host(host)
                 .port(port)
                 .build();
-            keyspace = new RandomStringGenerator.Builder().withinRange('a', 'z').build().generate(10);
-            clusterConfiguration = ClusterConfiguration.builder()
-                .host(Host.from(host, port))
-                .keyspace(keyspace)
-                .replicationFactor(1)
-                .maxRetry(10)
-                .minDelay(5000)
-                .build();
-            session = new FunctionRunnerWithRetry(MAX_RETRY).executeAndRetrieveObject(CassandraCluster.this::tryInitializeSession);
+            session = new SessionWithInitializedTablesFactory(
```

Comment:
How it is related to this PR ?

Reply:
Well, once I'm at it, CassandraCluster don't need to await anything, so, you know.... I consider usefull to stop lying to people.